lox_services/scraping/selenium_util.py

```diff
@@ -188,7 +218,7 @@ def wait_then_send_keys(
     - `input_text`: The text that will be sent to the element
     - `clear`: True if the field needs to be cleared before to get filled
     """
-    wait.until(lambda driver: driver.find_element(By.CSS_SELECTOR, selector))
+    wait.until(lambda dr: dr.find_element(By.CSS_SELECTOR, selector))
```

Comment:
please reverse this change

Reply:
Will do, but it does shadow a name - https://stackoverflow.com/questions/20125172/what-is-the-problem-with-shadowing-names-defined-in-outer-scopes